Tests/IndexStoreDBTests/IndexTests.swift

```diff
@@ -299,6 +299,34 @@ final class IndexTests: XCTestCase {
       IndexStoreDB.UnitIncludeEntry(sourcePath: main1, targetPath: uniq1, line: ws.testLoc("include_main1_uniq1").line),
     ])
   }
+    
+  func testFilesIncludedByFile() throws {
+    guard let ws = try staticTibsTestWorkspace(name: "MainFiles") else { return }
+    try ws.buildAndIndex()
+    let index = ws.index
+        
+    let main1 = ws.testLoc("main1").url.path
+    let uniq1 = ws.testLoc("uniq1").url.path
+    let shared = ws.testLoc("shared").url.path
+        
+    let includedFiles = index.filesIncludedByFile(path: main1)
+    XCTAssertEqual(includedFiles.count, 2)
+    XCTAssertEqual(includedFiles, [shared, uniq1])
+  }
+    
+  func testFilesIncludingFile() throws {
```

Comment:
Minor nitpick but these 2 tests are very similar, you could merge them into a single test invocation.

Reply:
done